lib/plugins/aws/provider/awsProvider.js

```diff
@@ -187,6 +187,13 @@ class AwsProvider {
     impl.addEnvironmentProfile(result, `AWS_${stageUpper}`);
 
     result.region = this.getRegion();
+
+    if (this.serverless.service.provider.deploymentBucketObject &&
+        this.serverless.service.provider.deploymentBucketObject.serverSideEncryption &&
+        this.serverless.service.provider.deploymentBucketObject.serverSideEncryption ===
+         'aws:kms') {
+      result.signatureVersion = 'v4';
+    }
```

Comment:
@reecefenwick @pmuens I added this code to handle the sigv4 parameter.

regarding the sdk version, it is set up as "^2.7.13" which pulls in the latest version 2.61.0, right? 
sorry if that is a stupid question or obviously wrong. This is really the first node project i have worked on.